analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -214,6 +170,59 @@ class Cargo(
             packageDir.startsWith(analysisRoot)
         } ?: false
 
+    private fun buildDependencyTree(
+        name: String,
+        version: String,
+        packages: Map<String, Package>,
+        metadata: JsonNode
```

Comment:
I believe it's fine to not repeat the type of the variable as part of the variable name, we're also not saying "versionString" here. That would only make sense if we had to deal with multiple metadata types in this context, like in `JsonNode` form and in a data class form.

Reply:
I think my comment was mis-leading, I actually did not mean to ask for the 'Node' suffix but to ask for consistency with other variable in that file because some do have such suffix.

Reply:
I agree with @sschuberth here.

Reply:
There is not comment relating to my argumentation but let's abort this minor discussion as you both agree.